### ISSUE TITLE
The "doctrine/cache" dependency is deprecated

### ISSUE THREAD

**Issue by jazithedev** [doctrine/doctrine2#8968]
Hello 👋,

The `doctrine/cache` package is deprecated and unmaintained, but it is still a strict dependency of the `doctrine/orm`. Is there a way to replace `doctrine/orm` with any other package, like for example `symfony/cache`? Are there any plans to remove that deprecated package from `doctrine/orm`?

All best!
Chris

**Comment by derrabus** [comment]
Yes, we need to get Doctrine Cache out of all interfaces eventually. For now, the recommendation is to use the only part of Doctrine Cache that is not deprecated (= the PSR-6 wrapper classes) and use Symfony Cache or whatever cache library your prefer.

Apart from that: PRs that phase out Doctrine Cache in favor of PSR-6 are more than welcome.
